packages/react-storage/src/components/StorageBrowser/components/PaginateButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { CLASS_BASE } from '../views/constants';
+import {
+  ButtonElement,
+  IconElement,
+  InputElementProps,
+} from '../context/elements';
+
+const BLOCK_NAME = `${CLASS_BASE}__paginate`;
+
+interface PaginateButtonProps extends InputElementProps {
```

Comment:
Nit:
* ${CLASS_BASE}__pagination
* PaginationButtonProps
* PaginationButton

Reply:
I updated the name to PaginationButton, but wouldn't the `BLOCK_NAME` use pagination-button rather than pagination?

Reply:
Oh yeah that's right